implementation/src/main/java/io/smallrye/faulttolerance/CompositeObservableCommand.java

```diff
@@ -80,40 +80,36 @@ protected CompositeObservableCommand(Callable<? extends CompletionStage<?>> call
         String metricsPrefix = MetricNames.metricsPrefix(operation.getMethod());
         // the retry metrics collection here mirrors the logic in HystrixCommandInterceptor.executeCommand
         // and MetricsCollectorFactory.MetricsCollectorImpl.beforeExecute/afterSuccess/onError
-        Observable<Object> observable = Observable.create(
-                subscriber -> {
-                    try {
-                        if (registry != null && retryContext != null && retryContext.hasBeenRetried()) {
-                            counterOf(metricsPrefix + MetricNames.RETRY_RETRIES_TOTAL).inc();
-                        }
-                        CompletionStage<?> stage = callable.call();
-                        if (stage == null) {
-                            subscriber.onError(new NullPointerException("A method that should return a CompletionStage returned null"));
+        Observable<Object> observable = Observable.create(subscriber -> {
```

Comment:
does the auto-formatter move the argument up?

Reply:
I did that myself, because otherwise the result #$%^ing horrible :-)

Reply:
for me the old form read much easier

Reply:
I personally much prefer

```java
Observable.create(subscriber -> {
    ...
});
```

over

```java
Observable.create(
    subscriber -> {
    }
);
```

but feel free to run the reformat locally to see the result with original code.